### DISCUSSION TITLE
Github Contribution chart issue

### DISCUSSION THREAD

**Discussion by DOodle25** [community/community#118702] [Other Feature Feedback, Questions, & Ideas] [answered]
hey i have a commit on future dates in a public repository even though i deleted previous commits and force pushed new repository to remove those previous commits its still showing one the contibution chart on a future date how can resolve this issue or contact github support mail
![help](https://github.com/community/community/assets/107390824/7bdc08b4-fa3f-4550-98c1-3f2c5f8ad501)

**Comment by DOodle25** [comment]
can someone atleast tell me how to contact github support any email??

**Reply by ebndev** [reply, marked as answer]
Hey [DOodle25](https://github.com/DOodle25)! Thank you for being a part of the GitHub community.

Are you still having this issue? I looked on your profile and didn't see the future commit on your contribution graph.

If you still would like an answer, we encourage you to use our search bar as many people have posted on this topic in the past. You can view some examples of posts [here](https://github.com/orgs/community/discussions?discussions_q=is%3Aopen+contribution+chart).

I’ve moved your post to our Profile 🖼 category, which is more appropriate for this type of discussion and may result in someone being able to answer your question.

For the future, if you do want to contact the support team, you can visit our [Support page](https://support.github.com/contact) and open a ticket with them :)

**Comment by AhmadRazaNewMan** [comment]
when i make i new repo and push the code it appear in github contribution and and only show there 2 contribution and when i want to push more code the reach the repo but it does not appear in the github repo on the same and on other  day as well

**Reply by DOodle25** [reply]
that should not happen perhaps your contribution chart is set to not show private repos but even though they should be visible to you in your own chart if such things keep happening you should contact [Github Support](https://support.github.com/contact)
